### ISSUE TITLE
Oracle: OracleException ["Unable to enlist in a distributed transaction"] thrown inside TransactionScope

### ISSUE THREAD

**Issue by bahram-aliyev** [linq2db/linq2db#181]
Oracle.DataAccess.Client.OracleException ["Unable to enlist in a distributed transaction"] exception thrown when DataConnection used inside TransactionScope context

**Comment by bahram-aliyev** [comment]
Thrown inside Connection property of DataConnection class ln 454 of master branch, when call _connection.Open();
if (_connection.State == ConnectionState.Closed)
{
        _connection.Open();
        _closeConnection = true;
}

**Comment by igor-tkachev** [comment]
Can you provide an example?

**Comment by bahram-aliyev** [comment]
At the beginning I would like to thank you for response.
Sure, for example
using(var txScope = new TransactionScope())
{
       using(var dbCtx = new FlowDbContext())
       {
               dbCtx.ExecuteFlow(0, 0, "tests", -1);
        }
        txScope.Complete();
}

I checked with two types of connection string configurations, both with the same result.

<add name="DefaultConnection" connectionString="DATA SOURCE=ora;PERSIST SECURITY INFO=True;USER ID=user;PASSWORD=user" providerName="Oracle" />

I don't think that it somehow affects on the current case, connection to db server established via VPN.

In addition, I tried similar example with MS SQL provider in the same circumstances, it worked well.
